kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/spi/datasource/JdbcQueryPropertiesExtension.java

```diff
@@ -31,7 +31,8 @@ public interface JdbcQueryPropertiesExtension extends TypedSPI {
     /**
      * Extend query properties.
      *
+     * @param props props
      * @return JDBC query properties for extension. Could NOT be null
      */
-    Properties extendQueryProperties();
+    Properties extendQueryProperties(Properties props);
```

Comment:
1, `Properties` result type should be removed.

2, Just extend properties of `props` parameter.

Reply:
And update all of the implementations